.github/workflows/static_checks.yml

```diff
@@ -128,3 +128,29 @@ jobs:
         if: always()
         run: |
           bazel run //scripts:kdoc_validity_check -- $(pwd) scripts/assets/kdoc_validity_exemptions.pb
+
+  third_party_dependencies_check:
+    name: Maven Dependencies Checks
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Bazel
+        uses: abhinavsingh/setup-bazel@v3
+        with:
+          version: 4.0.0
+
+      - name: Maven Repin Check
+        if: always()
+        run: |
+          REPIN=1 bazel run @unpinned_maven//:pin
+
+      - name: Maven Dependencies Update Check
+        if: always()
+        run: |
+          bazel run //scripts:maven_dependencies_list_check -- $(pwd) third_party/maven_install.json scripts/assets/maven_dependencies.pb
+
+      - name: License Details Check
+        if: always()
+        run: |
+          bazel run //scripts:generate_maven_dependencies_list -- $(pwd) third_party/maven_install.json scripts/assets/maven_dependencies.textproto scripts/assets/maven_dependencies.pb
```

Comment:
@BenHenning Confirmation regarding this CI check -
If a developer adds some dependencies and forgets to run the script then our CI check will fail and the developer will run the script to add all dependencies.
But if the developer runs the script and some manual work was needed but doesn’t complete it, then also our CI check would pass. We depend entirely on the reviewers to prevent this, however, if we also run the original script in CI then we can catch this case through CI.

Reply:
@BenHenning https://github.com/oppia/oppia-android/pull/3650/checks?check_run_id=3264829165 An example of a case where this check will be useful.

Reply:
Removed this as it is no longer needed.